static/fluent/en-US/main.ftl

```diff
@@ -631,7 +631,8 @@ transition-dialog-newsletter-label = Keep me updated with news about WebThings
 transition-dialog-agree-tos-label = Agree to the WebThings <a data-l10n-name="transition-dialog-privacy-policy-link">Privacy Policy</a> and <a data-l10n-name="transition-dialog-tos-link">Terms of Service</a>.
 transition-dialog-email =
     .placeholder = Email address
-transition-dialog-register = Register
+transition-dialog-register =
```

Comment:
You'll need to make the same change in all of the other translations, since quite a few have already been translated.

Reply:
Ah OK, I wasn't sure whether we were supposed to edit them all directly.